package.nls.json

```diff
@@ -384,6 +384,7 @@
     "DataScience.runAll": "Run all cells",
     "DataScience.runAbove": "Run cells above",
     "DataScience.runBelow": "Run cell and below",
+    "DataScience.toggleCellOutput": "Toggle output of all cells",
```

Comment:
I think this needs to be updated too, since we're only toggling output of selected cells now